modules/account-lib/src/coin/hbar/transactionBuilderFactory.ts

```diff
@@ -21,12 +22,20 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
     return this.initializeBuilder(tx, new TransferBuilder(this._coinConfig));
   }
 
+  /** @inheritDoc */
+  getTokenTransferBuilder(tx?: Transaction): TokenTransferBuilder {
+    return this.initializeBuilder(tx, new TokenTransferBuilder(this._coinConfig));
+  }
+
   /** @inheritDoc */
   from(raw: Uint8Array | string): TransactionBuilder {
     this.validateRawTransaction(raw);
     const tx = this.parseTransaction(raw);
     switch (tx.txBody.data) {
       case 'cryptoTransfer':
+        if (tx.txBody.cryptoTransfer && isTokenTransfer(tx.txBody.cryptoTransfer)) {
```

Comment:
Shouldn't TokenTransferBuilder be under case `tokenTransfer` and not `cryptoTransfer`.
I believe we need to update the hbar protobuf in account-lib to support these cases.